openmc/element.py

```diff
@@ -241,11 +235,21 @@ def expand(self, percent, percent_type, enrichment=None,
         # Interpret required enrichment as weight
         elif enrichment is not None and enrichment_target is not None:
 
-            # Check if is a single isotope mixture
+            # Provide more informative error message for U235
+            if enrichment_target == 'U235':
+                msg = "There is a special procedure for enrichment of U235 "\
+                      "in U. To invoke it do not specify neither "\
+                      "'enrichment_target' nor 'enrichment_type'. Provide "\
```

Comment:
I suggest a slight rewording of the second sentence:
"To invoke it, the arguments 'enrichment_target' and 'enrichment_type' should be omitted."